opencti-platform/opencti-front/src/private/components/common/stix_core_objects/StixCoreObjectKnowledgeBar.js

```diff
@@ -80,6 +80,18 @@ class StixCoreObjectKnowledgeBar extends Component {
             </ListItemIcon>
             <ListItemText primary={t('Overview')} />
           </MenuItem>
+          <MenuItem
```

Comment:
This menu is always available now ?
We have blank screen in some entity.
![image](https://user-images.githubusercontent.com/53513584/210725126-00bf5b68-5126-4f98-a568-b0b924fc27e8.png)

Reply:
Fixed